plugins/woocommerce/client/blocks/assets/js/utils/messages.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export const htmlElementMessages = {
```

Comment:
Should we consider adding a comment for this?

```suggestion
/**
 * Messages providing guidance for HTML element usage in block editor controls.
 * These messages help developers and users choose the appropriate semantic HTML elements
 * for their blocks.
 *
 * @example
 * ```tsx
 * <SelectControl
 *   value={ TagName }
 *   onChange={ (value) => setAttributes({ tagName: value }) }
 *   help={ htmlElementMessages[TagName] }
 * />
 * ```
 */
export const htmlElementMessages = {
```

Reply:
Sure happy to, thanks for the example :)